vllm/hpu/ops.py

```diff
@@ -49,33 +56,19 @@ def paged_attention_v1(query, key_cache, value_cache, head_mapping, scale, block
             .expand(batch_size, -1)
             .ge(context_lens.view(-1, 1))
             .view(batch_size, 1, 1, -1))
-    query.mul_(scale)
     query = query.unsqueeze(-2)
-    keys = fetch_from_cache(key_cache, block_tables, (0, 2, 3, 1))
     if query_heads != kv_heads:
         query = query.unflatten(1, (kv_heads, -1))
-        keys = [k.unflatten(1, (kv_heads, 1)) for k in keys]
         mask = mask.unsqueeze(2)
 
-    attn_weights = [torch.matmul(query, k) for k in keys]
-    attn_weights = (torch.cat(attn_weights, dim=-1)
-                    .masked_fill(mask, min_inf)
-                    .softmax(dim=-1))
-
-    values = fetch_from_cache(value_cache, block_tables, (0, 2, 1, 3))
-    if PA_SPLIT_VALUE:
```

Comment:
Woudn't it be better to remove the line 18 and 19 as variable PA_SPLIT_VALUE is no longer used?

Reply:
nice catch.